src/main/java/db/models/AirlineCompany.java

```diff
@@ -0,0 +1,33 @@
+package db.models;
+
+public class AirlineCompany {
+    private int companyId;
+    private String companyName;
+
+    public AirlineCompany(){}
+
+    public AirlineCompany(int company_id, String company_name){
+        this.companyId = company_id;
+        this.companyName = company_name;
+    }
```

Comment:
don't use underscore in Java variables names
```suggestion
    public AirlineCompany(int companyId, String companyName){
        this.companyId = companyId;
        this.companyName = companyName;
    }
```

Reply:
check all classes for the same issue